index.html

```diff
@@ -713,8 +696,10 @@ <h4>
       The negotiated communication carrier would then be used (separately) to
       perform certain activities between the two devices, such as sending photos
       to the other device, printing to a Bluetooth printer or streaming video to
-      a television set. Web NFC does not support this at the moment (see this
-      <a href="https://github.com/w3c/web-nfc/issues/364">issue</a>.
+      a television set.
+    </p>
+    <p class="issue" data-number="364">
+      Web NFC does not support this at the moment.
```

Comment:
As currently spec'ed, this is not supported.

Reply:
Done.